src/geo/ui/infowindow-model.js

```diff
@@ -1,31 +1,58 @@
 var _ = require('underscore');
 var Backbone = require('backbone');
 
-/**
- * Usage:
- * var infowindowModel = new InfowindowModel({
- *   template_name: 'infowindow_light',
- *   latlng: [72, -45],
- *   offset: [100, 10]
- * });
- */
+var defaultInfowindowTemplate = [
```

Comment:
Moved in [4138bdca9d15ad3d3f9f6085db6a97f2c5f12d86](https://github.com/CartoDB/cartodb.js/pull/1270/commits/4138bdca9d15ad3d3f9f6085db6a97f2c5f12d86).